tst/testbugfix/2024-10-11-alt-stab.tst

```diff
@@ -0,0 +1,7 @@
+gap> G := AlternatingGroup( 6 );;
+gap> hom1 := GroupHomomorphismByImages( G, G, [ (1,2,3,4,5), (4,5,6) ], [ (1,2,6,3,5), (1,4,5) ] );;
+gap> hom2 := GroupHomomorphismByImages( G, G, [ (1,2,3,4,5), (4,5,6) ], [ (1,2,3,4,5), (4,5,6) ] );;
+gap> tc := function ( g, h ) return (h^hom2)^-1 * g * h^hom1; end;;
+gap> stab := Stabilizer( G, One(G), tc );;
+gap> Size(stab);
+5
```

Comment:
Perhaps something along the lines of 
```
gap> Set( stab ) = Set( Filtered( G, g -> tc( One( G ), g ) = One( G ) ) );
true
```
would be safer? 

This would ensure the following case (which I only discovered just now, and which is fixed by this PR) is also caught:
```
gap> G := AlternatingGroup( 6 );;
gap> hom1 := GroupHomomorphismByImages( G, G, [ (1,2,3,4,5), (4,5,6) ], [ (1,2,6,3,5), (1,4,5) ] );;
gap> hom2 := IdentityMapping( G );;
gap> tc := function ( g, h ) return (h^hom2)^-1 * g * h^hom1; end;;
gap> stab := Stabilizer( G, One(G), tc );;
gap> Size(stab);
5
gap> Set( stab ) = Set( Filtered( G, g -> tc( One(G), g ) = One(G) ) );
false
```

Reply:
I've added your second test case, and made the checks more robust